internal/sherlock/routes.go

```diff
@@ -0,0 +1,34 @@
+package sherlock
+
+import (
```

Comment:
Similarly, I'm wondering if we should look at REST frameworks for Golang, I bet there are some good options out there

Reply:
😄